src/commands/config.ts

```diff
@@ -34,6 +35,14 @@ export function configCommand(program: Command) {
     .command('email <email>')
     .description('Set user email')
     .action(async (email: string) => {
+      if (cloudConfig.getApiKey()) {
+        logger.error(
+          "Cannot update email while logged in. Email is managed through 'promptfoo auth login'. Please use 'promptfoo auth logout' first if you want to use a different email.",
```

Comment:
error message may be too verbose 
```suggestion
          "Cannot update email while logged in. Email is managed through 'promptfoo auth login'.",
```